dags/analyzer_helper/tests/integration/test_discord_transform_raw_data.py

```diff
@@ -11,32 +11,32 @@ def setUp(self):
         self.platform_id = 'discord_platform'
         self.period = datetime(2023, 1, 1)
 
-    def test_transform_data_with_replied(self):
+    def tearDown(self):
+        self.collection.delete_many({})
+        self.client.close()
+
+    def test_transform_data_with_replied_user(self):
         raw_data = [
             {
                 "author": "user123",
-                "replied": "user789",
+                "replied_user": "user789",
                 "messageId": "msg123",
                 "channelId": "channel456",
-                "channelName": "general",
-                "threadId": "thread123",
-                "threadName": "discussion",
-                "isGeneratedByWebhook": False
+                "isGeneratedByWebhook": False,
+                "botActivity": False,
```

Comment:
We don't have the botActivity on rawinfos data. It would be identified via either the `isGeneratedByWebhook` or the guildmembers data

Reply:
Here's the data structure you could find 

https://rndadocs.notion.site/Analyzer-Discord-Pipeline-5009512d82a243deb35b3388e17b29c5#:~:text=rawinfos%20collection%20current%20data%20structure